### ISSUE TITLE
Add Layer Dropout

### ISSUE THREAD

**Issue by teamaffixmb-jake** [team-affix/aurora#55]
Dropout is a fundamentally important operation to prevent overfitting in neural networks.
